assets/js/theme/common/carousel/utils/arrowAriaLabling.js

```diff
@@ -1,26 +1,28 @@
-const NUMBER = '[NUMBER]';
-const integerRegExp = /[0-9]+/;
-const lastIntegerRegExp = /(\d+)(?!.*\d)/;
+export const updateTextWithLiveData = (textForChange, firstData, secondData) => {
```

Comment:
can we rename firstData and secondData args for them to be less abstract? For example, currentSlideNumber and actualSlideCount?

Reply:
changed with slideNumber and slideCount respectively.